mlflow/tracing/utils/__init__.py

```diff
@@ -6,28 +6,30 @@
 import logging
 import uuid
 from collections import Counter
+from contextlib import contextmanager
 from dataclasses import asdict, is_dataclass
 from functools import lru_cache
 from typing import TYPE_CHECKING, Any, Optional, Union
 
 from opentelemetry import trace as trace_api
 from packaging.version import Version
 
-import mlflow
-from mlflow.entities.span_status import SpanStatusCode
 from mlflow.exceptions import BAD_REQUEST, MlflowTracingException
 from mlflow.tracing.constant import SpanAttributeKey
 from mlflow.utils.mlflow_tags import IMMUTABLE_TAGS
+from mlflow.version import IS_TRACING_SDK_ONLY
 
 _logger = logging.getLogger(__name__)
 
 SPANS_COLUMN_NAME = "spans"
 
 if TYPE_CHECKING:
-    from mlflow.client import MlflowClient
     from mlflow.entities import LiveSpan
     from mlflow.types.chat import ChatMessage, ChatTool
 
+    if not IS_TRACING_SDK_ONLY:
+        from mlflow.pyfunc.context import Context
```

Comment:
```suggestion
    from mlflow.pyfunc.context import Context
```

Do we need `IS_TRACING_SDK_ONLY` guard? This block is ignored at runtime.

Reply:
Not really, validated it doesn't affect.